python/cudf/cudf/core/index.py

```diff
@@ -579,7 +579,7 @@ def any(self):
         """
         return self._values.any()
 
-    def to_pandas(self):
+    def to_pandas(self, nullable=False):
```

Comment:
For some of these index `to_pandas` functions for which `nullable` has no effect, I suggest just `**kwargs`.